packages/client/src/helpers/fetchApi.js

```diff
@@ -11,6 +11,23 @@ function getDefaultHeaders() {
   return headers;
 }
 
+/*
+* Turns param object {limit: 1, paginateFrom: 2} into uri string '?limit=1&paginateFrom=2
+*/
+export const serializeQuery = (params = {}) => {
+  const pairs = [];
+
+  Object.entries(params).forEach(([key, value]) => {
+    if (value !== undefined) {
+      pairs.push(
+        `${encodeURIComponent(key)}=${encodeURIComponent(value)}`,
+      );
+    }
+  });
+
+  return pairs.length ? `?${pairs.join('&')}` : '';
+};
+
```

Comment:
I think it might be best to use native way of doing this in JS:
```js
export const serializeQuery = (params = {}) => `?${new URLSearchParams(params).toString()}`;
```
Note that the above will return a string of `'?'` when `params` is empty; it should be perfectly fine to have a dangling `?` for this use-case, but could be modified to match the behavior that you currently have implemented:
```js
export const serializeQuery = (params = {}) => {
  const searchParams = new URLSearchParams(params);
  return searchParams.size ? `?${searchParams.toString()}` : '';
}
```

Reply:
updated to include native approach w/ minor adjustments to remove empty values